birdhouse/components/cowbird/default.env

```diff
@@ -34,5 +34,12 @@ export COWBIRD_MONGODB_SERVICE=mongodb
 export COWBIRD_MONGODB_HOST=${COWBIRD_MONGODB_SERVICE}
 export COWBIRD_MONGODB_PORT=27017
 
+# this dependency is only required if COWBIRD_MONGODB_SERVICE is the one provided in config/mongodb.
+# (include this by default to support backwards compatibility for now)
+COMPONENT_DEPENDENCIES="
+  $COMPONENT_DEPENDENCIES
+  ./config/mongodb
```

Comment:
The problem is that it *can* use its own mongodb instance but by default it uses the one in config/mongodb.

I didn' t include it in the previous PR because it's not technically a dependency and it would break backwards compatibility because it would require the user to update their env.local files to include config/mongodb explicitly.

This PR is not backwards compatible so I put it in here but honestly I don't love this workaround. I think I'm going to change this for this PR too. Let me think about how to do this better and I'll update it shortly.

Reply:
> it would require the user to update their env.local files to include config/mongodb explicitly

But this change fixed that and ensure no manual edit of `env.local` is required, which make this change back-compat.  The previous PR is actually not back-compat if Cowbird was activated alone.

During testing, we did not find out because probably another component pulled `./config/mongodb` for Cowbird to use.

I would keep this as-is for now, unless you change the way Cowbird "ask for mongodb".  This is more backward compat than the previous PR :)

Reply:
Right.... though the previous PR was backwards compatible because the assumption was that every deployment was using all components in the original docker-compose.yml file (ie. the "default components") which included mongodb.

So yes, it would break if cowbird was activated alone but there would be no existing deployment that would ever do that.